index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="./scripts/script.js" defer type="module"></script>
+    <link rel="stylesheet" href="./SCSS/style.css" />
+    <title>Google Book API Search Engine</title>
+  </head>
+  <body>
+    <header class="home">
+      <h1 class="home__heading">FIND YOUR BOOK...</h1>
+
+      <input
+        class="home__search"
+        id="searchBook"
+        type="text"
+        placeholder="Search book to read"
+      />
+      <button class="home__button" id="getData">SEARCH BOOK</button>
+    </header>
+    <h2 id="showResult"></h2>
+    <section></section>
+  </body>
+</html>
```

Comment:
add a paragraph that explains what this page is about/how to use, just so users have a full understanding to what they can do on this page